homeassistant/components/wink/sensor.py

```diff
@@ -87,3 +86,15 @@ def state(self):
     def unit_of_measurement(self):
         """Return the unit of measurement of this entity, if any."""
         return self._unit_of_measurement
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        super_attrs = super().device_state_attributes
+        _LOGGER.debug("Adding in eggs if egg minder")
```

Comment:
We shouldn't have any side effects, eg logging, in state properties.

Reply:
Oh okay probably should have took that out anyway. Should I open another PR to remove it or ?

Reply:
Yes, please! :+1: